spec/requests/errors_spec.rb

```diff
@@ -39,16 +39,16 @@
     end
   end
 
-  describe "bypassing user access checks" do
+  describe "GET /any-path-for-a-document" do
     it "returns a not found response when a document doesn't exist" do
       get document_path("document-that-does-not-exist")
 
       expect(response).to have_http_status(:not_found)
     end
   end
 
-  describe "local data unavailable" do
-    it "returns service unavailable" do
+  describe "ANY /document-path-using-local-data" do
+    it "returns service unavailable when locale data is unavailable" do
```

Comment:
Should this be "local" rather than "locale"?

Reply:
Oops - updated 👍.